k8s/agones/chart/agones-mp/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
```

Comment:
(I still think that agones helm chart should be check-in unpacked, currently I cannot see changes between `0.8.1` and `0.10.0` without downloading the archives.)